tls/cfsslscan_common.go

```diff
@@ -1,36 +1,105 @@
 package tls
 
+import "fmt"
+
+type hashAlgID uint8
+
 const (
-	hNone uint8 = iota
-	hMD5
-	hSHA1
-	hSHA224
-	hSHA256
-	hSHA384
-	hSHA512
+	HashNone hashAlgID = iota
+	HashMD5
+	HashSHA1
+	HashSHA224
+	HashSHA256
+	HashSHA384
+	HashSHA512
 )
 
+func (h hashAlgID) String() string {
+	switch h {
+	case HashNone:
+		return "None"
+	case HashMD5:
+		return "MD5"
+	case HashSHA1:
+		return "SHA1"
+	case HashSHA224:
+		return "SHA224"
+	case HashSHA256:
+		return "SHA256"
+	case HashSHA384:
+		return "SHA384"
+	case HashSHA512:
+		return "SHA512"
+	default:
+		return "Unknown"
+	}
+}
+
+type sigAlgID uint8
+
 // Signature algorithms for TLS 1.2 (See RFC 5246, section A.4.1)
 const (
-	sigAnon uint8 = iota
-	sigDSA
-	sigRSA
-	sigECDSA
+	SigAnon sigAlgID = iota
+	SigDSA
```

Comment:
RSA first, DSA second. I was having a pull request to fix this before, but since it doesn't matter, because we include them all.